DINT-1585: Custom context is not supported by contextManager (#160)
adobe/commerce-events: examples/events/custom-event-override.md

    ],
});
```

### Example 2 - adding custom context before publishing event

```javascript
const mse = window.magentoStorefrontEvents;

mse.context.setCustom({
  productListItems: [
    {
      productCategories: [
        {
          categoryID: "cat_15",
          categoryName: "summer pants",
          categoryPath: "pants/mens/summer",
        },
      ],
    },
  ],
});

mse.publish.productPageView();
```

> ### Example 3 - custom context set in publisher will overwrite custom context set in ACDL before.

Reviewer: ```suggestion
### Example 3 - the custom context set in the publisher overwrites the custom context previously set in ACDL.
```

> In this example page view event will have **Custom Page Name 2** in `web.webPageDetails.name` field

Reviewer: ```suggestion
In this example, the `pageView` event will have **Custom Page Name 2** in the `web.webPageDetails.name` field
```
